cmd/kubelet/app/server.go

```diff
@@ -201,6 +203,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.CloudProvider, "cloud_provider", s.CloudProvider, "The provider for cloud services.  Empty string for no provider.")
 	fs.StringVar(&s.CloudConfigFile, "cloud_config", s.CloudConfigFile, "The path to the cloud provider configuration file.  Empty string for no configuration file.")
 	fs.StringVar(&s.ResourceContainer, "resource_container", s.ResourceContainer, "Absolute name of the resource-only container to create and run the Kubelet in (Default: /kubelet).")
+	fs.StringVar(&s.CgroupParent, "cgroup_parent", s.CgroupParent, "Optional parent cgroup to pass to Docker. All pods will run underneath it.")
```

Comment:
For this specifically, we'll probably have the runtime be able to handle this as well.